README.md

```diff
@@ -1,149 +1,24 @@
-```
-
-Using this inventory file one can deploy a Kubernetes or OpenShift cluster.
-
-### Kubernetes cluster
-
-
-```bash
-$ ansible-playbook -i inventory playbooks/cluster/kubernetes/config.yml
-```
-
-### OpenShift cluster
-
-
-Be sure that you have enough space on your machines for docker storage and
-modify [defaults values][docker-storage-setup-defaults] accordingly.
-Follow [docker-storage-setup] documentation for more details.
-
-
-```bash
-$ ansible-playbook -i inventory \
-    -e "openshift_ansible_dir=$PWD/openshift-ansible/ \
-    openshift_playbook_path=playbooks/byo/config.yml \
-    openshift_ver=3.7" playbooks/cluster/openshift/config.yml
-```
-where
-* `openshift_ver` specifies what version of OpenShift one wants to deploy. Choose from
-  * 3.7
-  * 3.9
-* `openshift_ansible_dir` is a path to a cloned [OpenShift Ansible][openshift-ansible-project] git repository.
-* `openshift_playbook_path` is a path to OpenShift deploy playbook in [OpenShift Ansible][openshift-ansible-project]. Choose from
-  * `playbooks/byo/config.yml` for OpenShift 3.7 (default)
-  * `playbooks/deploy_cluster.yml` for OpenShift 3.9
-
-## Install KubeVirt on an existing cluster
-
-### Kubernetes cluster
-
-Currently we don't have a playbook which installs KubeVirt on a Kubernetes cluster.
-
-### OpenShift cluster
-
-Install KubeVirt on your OpenShift cluster.
-
-```bash
-$ ansible-playbook -i localhost playbooks/kubevirt.yml
-```
-
-See [KubeVirt parameters documentation](./roles/kubevirt/README.md) for more details on parameters.
-
-## Deploy a new Kubernetes or OpenShift cluster and KubeVirt with Lago
-
-This section describes how to deploy a KubeVirt or OpenShift cluster with KubeVirt installed on it using Lago.
-
-Following example is executing top level playbook `playbooks/automation/check-patch.yml`.
-
-```bash
-$ ansible-playbook -i inventory \
-    -e "cluster_type=openshift \
-    provider=lago \
-    inventory_file=inventory \
-    openshift_ver=3.7 \
-    openshift_playbook_path=playbooks/byo/config.yml \
-    openshift_ansible_dir=openshift-ansible/" \
-    playbooks/automation/check-patch.yml
-```
-
-where
-* `cluster_type=kubernetes|openshift` defines a desired cluster type,
-* `openshift_ver` specifies what version of OpenShift one wants to deploy. Choose from
-  * 3.7
-  * 3.9
-* `openshift_ansible_dir` is a path to a cloned [OpenShift Ansible][openshift-ansible-project] git repository,
-* `openshift_playbook_path` is a path to OpenShift deploy playbook in [OpenShift Ansible][openshift-ansible-project]. Choose from
-  * `playbooks/byo/config.yml` for OpenShift 3.7 (default)
-  * `playbooks/deploy_cluster.yml` for OpenShift 3.9
-* `provider` defines provider. At the moment, `lago` is the only supported provider,
-* `inventory_file` is a path to the inventory file. Lago adds VMs into the inventory file once they are created, so it needs to know where it is located.
+See the [playbooks readme file](playbooks/README.md) for details.
```

Comment:
This greatly simplifies the README, but in my opinion the command to run kubevirt on existing cluster should remain here.  It's the bare minimum you need to getting started with kubevirt and I think most folks will only need to read this far.  What do you think about keeping it here?


```
## Quickstart
Deploy KubeVirt on an existing OpenShift cluster.

ansible-playbook -i localhost playbooks/kubevirt.yml

```